tests/contrib/django/asgi.py

```diff
@@ -1,4 +1,28 @@
+import os
+
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter
+from channels.routing import URLRouter
 from django.core.asgi import get_asgi_application
+from django.urls import re_path
+
+
+# Application must be run in the project root to find this settings ex. ddtrace/
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.contrib.django.django_app.settings")
```

Comment:
is this too late for this to be set?

Reply:
I think it should be fine. This is consistent with how the customer set their application: https://github.com/DataDog/dd-trace-py/issues/2945#issuecomment-951471462